client/package.json

```diff
@@ -83,6 +83,7 @@
     "react-dnd": "^2.4.0",
     "react-dnd-html5-backend": "^2.4.1",
     "react-dom": "^15.5.4",
+    "react-dropzone": "^3.13.1",
```

Comment:
So just add this to the webpack config under the lib section? That will indicate to webpack to package it?